tests/reftests/conflict-resto.test

```diff
@@ -0,0 +1,416 @@
+a5d7cdc0c914
```

Comment:
It might be simpler and less costly to run to just have the simple generic case described in https://github.com/ocaml/opam/issues/4373#issuecomment-1006121785 instead

Reply:
Indeed, and that would be interesting as well; I'd say "real-world" cases have value too, and the cost of extracting the opam root is only for the first run though